pkg/scheduler/core/generic_scheduler.go

```diff
@@ -389,12 +394,21 @@ func (g *genericScheduler) getLowerPriorityNominatedPods(pod *v1.Pod, nodeName s
 
 // numFeasibleNodesToFind returns the number of feasible nodes that once found, the scheduler stops
 // its search for more feasible nodes.
-func (g *genericScheduler) numFeasibleNodesToFind(numAllNodes int32) int32 {
-	if numAllNodes < minFeasibleNodesToFind || g.percentageOfNodesToScore <= 0 ||
-		g.percentageOfNodesToScore >= 100 {
+func (g *genericScheduler) numFeasibleNodesToFind(numAllNodes int32) (numNodes int32) {
+	if numAllNodes < minFeasibleNodesToFind || g.percentageOfNodesToScore >= 100 {
 		return numAllNodes
 	}
-	numNodes := numAllNodes * g.percentageOfNodesToScore / 100
+
+	if g.percentageOfNodesToScore <= 0 {
```

Comment:
Do we need a way to disable this mechanism? Is it meaningful for end users?

Reply:
The current PR solves the scenario where the user does not specify the percentage. If the user has already specified, the mechanism will not be triggered. When the user chooses to set the percentage, it is equivalent to being disabled.